source/transactions/transactions.rst

```diff
@@ -810,26 +810,30 @@ certain retries of commitTransaction and abortTransaction) to the same mongos.
 When to unpin
 ^^^^^^^^^^^^^
 
-Drivers MUST unpin a ClientSession when a command within a transaction,
-including commitTransaction and abortTransaction, fails with a
-TransientTransactionError. Transient errors indicate that the transaction
-in question has already been aborted or that the pinned mongos is
+Drivers MUST unpin a ClientSession in the following situations:
+
+#. The transaction is aborted. The session MUST be unpinned regardless of
```

Comment:
The numbered list here does not render correctly: https://github.com/mongodb/specifications/blob/fc561d9216774fba026bf210f283a5881d09b373/source/transactions/transactions.rst#when-to-unpin

The issue is that items that span multiple lines need to be indented: https://docutils.sourceforge.io/docs/user/rst/quickref.html#enumerated-lists

Reply:
Sorry, I meant to check the rendering when I first put up the PR but forgot.